addons/l10n_it_ddt/models/stock_picking.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    transport_reason_id = fields.Selection([('sale', 'Sale'), ('repair', 'Repair')], string='Transport Reason')
+    transport_method_id = fields.Selection([('mittente', 'MITTENTE'), ('desstinatario', 'DESTINATARIO'), ('corriere', 'CORRIERE')], string='Transport Reason')
+    parcels = fields.Integer(string="parcels")
+    volume = fields.Integer(string="volume")
+    size = fields.Text(string="size")
```

Comment:
This fields are localizations ones shouldn't they be l10n_**?

Reply:
you are right

Reply:
it done.